tst/testrandom.g

```diff
@@ -49,8 +53,22 @@ randomTest := function(collection, method, checkin...)
     fi;
 end;;
 
+randomTest := function(collection, method, checkin...)
+    local checkmethod;
+    if Length(checkin) = 0 then
+        checkmethod := \in;
+    else
+        checkmethod := checkin[1];
+    fi;
+
+    randomTestInner(IsMersenneTwister, GlobalMersenneTwister, x -> method(x), method, collection, checkmethod);
+    randomTestInner(IsGAPRandomSource, GlobalRandomSource, x -> method(GlobalRandomSource, x), method, collection, checkmethod);
```

Comment:
Why do we once `x -> method(x)` and once `x -> method(GlobalRandomSource, x)`? Aha, I assume there is an implicit assumption is that if no RS is specified, then `GlobalMersenneTwister` is used? The manual only says this (emphasis mine):
> *Most* methods for `Random` in the GAP library use the `GlobalMersenneTwister` as random source.

I am happy to enforce this here in the testing code, but I think this should be stated explicitly, at least in a comment right here, e.g. like this:
> If no random source is specified, functions producing randomness are expected to default to using <GlobalMersenneTwister>

Reply:
I think we should possibly strengthen the documentation, to "All methods", I believe the only method which uses anything else is the undocumented RANDOM_LIST.